kubernetes/apply-default.sh

```diff
@@ -1,63 +1,63 @@
 #!/bin/sh
 set -eu
 
-ROOT_DIR=$(
-    cd $(dirname -- $0)
+ROOT_DIR="$(
+    cd "$(dirname -- '$0')"
```

Comment:
Codacy found an issue: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5892274)